cluster/network.go

```diff
@@ -435,14 +431,10 @@ func checkPlaneTCPPortsFromHost(ctx context.Context, host *hosts.Host, portList
 		return err
 	}
 
-	clogs, err := docker.ReadContainerLogs(ctx, host.DClient, PortCheckContainer, true, "all")
-	if err != nil {
-		return err
+	containerLog, logsErr := docker.GetContainerLogsStdoutStderr(ctx, host.DClient, PortCheckContainer, "1")
+	if logsErr != nil {
+		logrus.Debugf("[network] Failed to get network port check logs: %v", logsErr)
```

Comment:
should we log it in warn? you won't be able to see it on user's setup where the log level is info